plugins/content/tjreportsfields/tjreportsfields.php

```diff
@@ -0,0 +1,289 @@
+<?php
+/**
+ * @package     TJReports
+ * @subpackage  com_tjreports.tjreportsfields
+ *
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Class for Tjreportsfields Content Plugin
+ *
+ * @since  1.1.0
+ */
+class PlgContentTjreportsfields extends JPlugin
+{
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 *
+	 * @since  3.2.11
+	 */
+	protected $autoloadLanguage = true;
+
+	protected $customFieldsTable;
+
+	protected $customFieldBeingEdited;
+
+	protected $unSupportedFields = array ('imagelist', 'media', 'repeatable', 'sql');
+
+	protected $fieldTypeToColumnTypeMapping = array (
+		'calendar'      => 'datetime',
+		'checkboxes'    => 'varchar(255)',
+		'color'         => 'varchar(7)',
+		'editor'        => 'text',
+		'integer'       => 'int(11)',
+		'list'          => 'varchar(255)',
+		'radio'         => 'varchar(255)',
+		'text'          => 'text',
+		'textarea'      => 'text',
+		'url'           => 'varchar(250)',
+		'user'          => 'varchar(400)',
```

Comment:
This can be int (11) ?

Reply:
This is not userid - this is username, which is what we are saving